### ISSUE TITLE
Maximo User Process Discovery - Pavement Ops (Sealcoat)

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#16928]
## Objective 
The Data & Technology Services Division is initiating a series of meetings to learn more about Maximo user experiences and processes in TPW. In this meeting we'll be asking questions about how individual teams use Maximo and how it fits into their work processes.

## Goals:  
- Maximo system performance and functionality 
- User experience and interface experiences 
- Current processes within and related to Maximo 
- User feedback and improvement  
- Future roadmap and upgrades 

### User experience and interface experiences 
- Explain your process on how you use Maximo? 

### PROCESS - Planning
- Who plans the work? How do you get work to do? 
- How is work scheduled and issued out 
 
### PROCESS - Implementation
- How is work scheduled and issued out?
- Who is executing the implementation? 
 

### PROCESS - Field Work
- Who is executing the work - implementation? 
- Who is reporting on the work and what are they tracking? 

#### Maximo system performance and functionality 
- How long is it taking to do a Maximo work order in the field? 

### PROCESS - Review & Reporting
- What is your final review, QC process? 
- What reporting do you care about? How are you pulling that information out? 
- What are you reporting? 

### User feedback and improvement  
- What challenges are you having with Maximo? 

### Future roadmap and upgrades 
- What would you like your work order management system to do that you’re not getting right now? What’s missing? 

### Points of Contacts 

------
- [x] Schedule meeting 
- [x] Meet and take notes
- [x] Process notes on Miro [Maximo Assessment Plan](https://miro.com/app/board/uXjVN7sxj9I=/)

**Comment by ChristinaTremel** [comment]
# 4/23/2024 Notes
### User experience and interface experiences
- Explain your process on how you use Maximo?

### PROCESS - Planning
- Who plans the work? How do you get work to do?
   - Receive work through the Service Plan
   - Everything is generated for them and imported into Maximo
   - They have 500ish work orders, but there are 3 child work orders within those work orders
      - Sweeping
      - Crack Seal
   - Not spontaneous 311 work or anything
   - Charge parts and labor and close out the child before the parent
   - Sealcoat season is about 5 months and everything else is prep for the next season
   - They have 2 teams, overlay and sealcoat
      - Within the sealcoat, there are multiple crews
         - Crack seal is 7-9 people
         - Hot mix is 5 people
         - Sealcoat is 21 people
   - Sealcoat address fogseal
         - 1 sealcoat team and 1 fogseal team. Usually 2 sealcoat teams but they are understaffed.
- How is work scheduled and issued out
   - Decide on if they want to start north or south and start the crack sealing first
   - After crack seal is when the sealcoat can occur
   - Schedule and synchronize in order. Work from the furthest out towards the stock pile/yard.
      - Scheduling is completed by the superintendent and the supervisors together
   - The schedule is managed on a spreadsheet and manually update the spreadsheet when it’s been completed.
- How to the coordinators come into play - this is the first year of having coordinators
   - Andre’s role is to handle all of the permitting. If they come into complications on work that can’t be done, if the guys aren’t notified directly by OCE they are notified by the coordinators and will defer the work to the next maintenance cycle
- Most streets they work on is residential and those permits are very straight forward
   - For streets that are more complicated, roundabouts, PHBs, etc they have to create the engineering plans.
   - The city doesn’t have a definition of what seal coat work is because it’s fast moving but can take a while when they have to go one lane at a time.
   - They have to get creative or pay for plans if they want to have the work permitted
   - Will be using an outside engineering firm to create the engineering TCP
      - This process is long and drawn out
      - The crews complete 10 projects per day so they move pretty fast and this is definitely going to slow down the process
      - Started in mid Feb but they are still waiting on fully submitting the proposal. That throws off the whole process. For overlay, they were at $250,000 and they’ve cut it for $50,000 because the timeline is so drawn out and is out of scope for this FY.
   - Were given a secondary permit last year, but didn’t get it for this year and they will have to be primary on all permits.
      - Everything was the same as last year, but it didn’t get approved this year for some reason
   - Tracking permitting - uses the permitting system in ABC portal

### PROCESS - Implementation
- How is work scheduled and issued out?
- Who is executing the implementation?

### PROCESS - Field Work
- Who is executing the work - implementation?
- Who is reporting on the work and what are they tracking?
   - Leads are going to be in charge of submitting the information in Maximo
      - Previously Supervisors were doing this
   - They enter details into Maximo the next day because there are so many streets completed in one day. Whatever they do today, they enter into Maximo tomorrow.
      - The guys have paper form log tracking sheets to note how much materials, specs, and labor were completed on the work order.
      - They currently fill out the work order in the office at the beginning of the morning
   - John F noted that they are trying to get to crews completing the work tickets in the field so they have more real time information. The crews now have a laptop and iPad
- Supervisors and Leads have a laptop and iPad
   - They have received 1 or 2 iPads and are waiting on a few more
   - Jonathan created a ticket today for 2 more iPad for sealcoat and fogseal **#16923**
   - To get everyone in Pavement Ops an iPad, they will probably need 6 total (4 additional tablets for Overlay)

### Maximo system performance and functionality

- How long is it taking to do a Maximo work order in the field?
   - 15-20 minutes to fill out a work order in Maximo
   - 7-10 work orders in a day based on how long the road is

### PROCESS - Review & Reporting
- What is your final review, QC process?
   - QC process is completed by the Supervisors on 4COMP
   - QC process is completed by Superintendent on 4QACOMP
   - QC 4COMP and 4QACOMP work orders
      - It’s a lengthy process because they are checking all the specs and materials
         - Need to make sure to get it right because it’s what Dave takes to management.
   - Closes at 5COMP after 180 days
   - Inspector has only been in the position for 4 months but he’s going to be checking the roads before work is done to identify any issues and any level ups that need to happen
        - He also inspects for any trees in the ROW that will obstruct equipment
        - Inspects the streets and if there’s any issues, Jonathan goes into Maximo and puts the work order on hold and adds details on why he’s putting it on hold
        - If there’s no issues, updates status to 2APR
- What reporting do you care about? How are you pulling that information out?
- What are you reporting?
    - Reporting is handled by Dave A. Some comes from Maximo and some comes from records that they keep - some of this is due to the limitations of Maximo
    - Ton used, lane miles, projects completed
    - Also report out on sealcoat for level ups (in the specification). Level ups are the prep work that happens before the season. That’s a number that they put in the specification
        - Will also start reporting on square footage.
    - Also independently of Maximo they track bad weather days for bad weather and special events (seal coat and fog seal is impacted, but overlay is not affected as much)
        - Tracked on a spreadsheet
        - If the streets are wet, sealcoat can’t happen
        - They have a gauge if the rain chance is 50% or more, the sealcoat operation is shut down
    - Things they can’t track
        - Projects will be extended out and since the work order was already created, they have predicted lane miles but if they are extended or shortened that will throw off the projections
        - If street was in too bad or repair but if it’s part of the projection, they can’t adjust the projection
        - They can’t adjust the projection

### User feedback and improvement

- What challenges are you having with Maximo?
    - If we could work on a faster way of reporting, that would be great.
        - It’s going to be hard to keep up with the pace of Maximo
        - They have to take pics to close out the work order
        - Would like to have a faster way of getting this information into Maximo
    - Easier reporting
        - Have certain reports they can run, but if they have specialized reports they have to contact DTS.
        - Would love to be able to generate their own reports
        - Reporting on various specification
    - Cleaning up specifications. There are several specification that they don’t use and they could clean a lot of them up to be exactly what the crews use
        - Organize specifications by what the crews use
    - Would be nice for the Superintendent and Supervisor to be able to determine what’s required to close a work order. Would be nice to set those restrictions in house for the independent work crew.
    - Service plans are all managed by spreadsheet
        - Load service plan and allow the service plan to be flexible
        - If it’s scheduled for 1/1/25 and it wasn’t closed out until 2 days later, he would love to see that the next project is shifted out
        - This is what stops them from wanting to put the project into Maximo
    - They currently don’t have access to the spatial tab in Maximo and would like to see it because it could be very useful for reporting.

**Comment by rebekkabry** [comment]
[High Level Work Order Phasing Visual ](https://miro.com/app/board/uXjVKI0wi4c=/)

![image](https://github.com/cityofaustin/atd-data-tech/assets/134445095/9f43eaa8-7e13-4400-a911-54c5b85f966f)

**Comment by ChristinaTremel** [comment]
Met today and added notes to the Miro board!
